Add optional tweaking of the .gv output (#215)
formatc1702/WireViz: docs/syntax.md

  manufacturer: <str>  # manufacturer name  
```

> ## Tweak entries

Reviewer: ```suggestion
## GraphViz tweaking (experimental)
```

```yaml
  # Optional tweaking of the .gv output.
  # This feature is experimental and might change
  # or be removed in future versions.

  override:  # dict of .gv entries to override
    # Each entry is identified by its leading string
    # in lines beginning with a TAB character.
    # The leading string might be in "quotes" in 
    # the .gv output. This leading string must be
    # followed by attributes in [square brackets].
    # Entries containing HTML in an attribute are
    # not supported.
    <str>:  # leading string of .gv entry
      <str> : <str>  # attribute and its new value
      # Any number of attributes can be overridden
      # for each entry

>   append:  # single or list of .gv entries to append
>     <str>  # strings to append might have multiple lines

Reviewer: ```suggestion
  append: <str/list> # string or list of strings to append to the .gv file
```
